index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=crete+round">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=poppins">
+<body>
+    <header class="header">
+        <div class="top">
+            <div class="time">
+                <p>08:45</p>
+            </div>
+            <div class="icons">
+                <i class="fa fa-wifi"></i>
+                <i class="fa fa-battery-full"></i>
+                <i class="fa fa-signal"></i>
+            </div>
+        </div>
+        <div class="main-header">
+            <h2 class="name">Kenny Egun</h2>
```

Comment:
- [ ] This is supposed **to be a heading but also an anchor tag that is always directing the user to the home page**. This is a convention of web designs. Please **wrap the heading text inside an `<a>` tag and then style it to look similar as it is in the Figma design you chose 😉**